docs/use_cases/node_representation_learning.md

```diff
@@ -0,0 +1,269 @@
+# POST2 Leveraging Relational Information
+
+## Introduction
+
+Different types of information, like words, pictures, and connections between things, show us different sides of the world. Relationships, especially, are interesting because they show how things interact and create networks. In this post, we'll talk about how we can use these relationships to understand and describe things in a network better.
+
+We're diving into a real-life example to explain how entities can be turned into vectors using their connections, a common practice in machine learning. The dataset we're going to work with is the a subset of the Cora citation network. It comprises 2708 scientific papers (nodes) and the connections indicate citations between them. Each paper has a BoW (Bag-of-Words) descriptor containing 1433 words. The challenge at hand involves predicting the specific scientific category to which each paper belongs to, selecting from a pool of seven distinct categories.
```

Comment:
I'd propose a descriptive statistic about the dataset. Let's calculate cosine similarity between all items, and create a chart that shows:
- bins of cosine similarity ranges in terms of BoW representations (1-0.98, 0.98-0.96, etc.)
- against the probability (or just counts of pairs having or not having a citation connection on a 2 bidirectional barchart like [this](https://images.app.goo.gl/vqnkReDQFFm3bARf9) of having a citation between them)
This would show how connected the 2 aspects are, how much information is there in incorporating both aspects into our vectors.

Reply:
![bow_cos](https://github.com/superlinked/VectorHub/assets/14368841/a203a2c6-7062-4db7-8c5e-6ceeb77dd5b9)

Reply:
This is the distribution of the pairwise cosine similarities.

Reply:
*In the second bullet point do you want to show how well the cosine similarities reflect connections in the graph? 
I don't exactly get it how the plot should look like. 
Additionally I can visualize the ROC curce of nodes being connected predicted based on BoW feature cosine similarity - that would tell us something like *.

Reply:
Added this part in the latest commit. For me it feels a bit odd, we should tell the reader why we need this statistic. Do you have any idea how to blend it in more to the "story line"?